SwiftSocket.podspec

```diff
@@ -19,7 +19,7 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = '10.10'
   s.tvos.deployment_target = '9.0'
   s.source   = {
-    :git => 'https://github.com/swiftsocket/SwiftSocket.git',
+    :git => 'https://github.com/dinobei/SwiftSocket.git',
```

Comment:
You probably don't want to include this change in the PR.

Reply:
@keab42  Oh, I did not know. That is a mistake. PullRequest corrected.